index.html

```diff
@@ -3,8 +3,15 @@
 <head>
   <meta charset="utf-8">
   <meta name="viewport" content="width=device-width, initial-scale=1">
-  <link rel="stylesheet" href="style.css">
+  <link rel="stylesheet" href="styles/style.css">
   <title>David Porter</title>
+  <!-- <script id="neighborhood-template" type="text/x-handlebars-template">
+    <h2>{{name}}</h2>
+    <p>City of {{city}}</p>
+    <p>Population of {{population}}</p>
+  </script>
+ -->
```

Comment:
Once you load the Handlebars library in here, uncomment this code and replace it with the properties for your objects.